tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -61,18 +65,23 @@ def main(config, parsed_args, cloud_logger):
     sts_client = discovery.build(
         'storagetransfer', 'v1', credentials=config.target_project_credentials)
 
+    try:
+        retain_source_bucket = 'False' if (len(config.retain_source_bucket) == 0) else config.retain_source_bucket
```

Comment:
Entire block is redundant since retain_source_bucket can be directly accessed  using config.retain_source_bucket and will be a bool value (True/False)